src/layer/__tests__/S2L1CLayer.ts

```diff
@@ -1,6 +1,7 @@
-import 'jest-setup';
+import '../../../jest-setup';
 
-import { ApiType, S2L1CLayer } from 'src';
+import { S2L1CLayer } from '../S2L1CLayer';
+import { ApiType } from '../const';
```

Comment:
Could these both be imported like in `src/layer/__tests__/S1GRDAWSLayer.ts` or others - `import {...} from '../../index';`?

Reply:
Changed

Reply:
Resolving